src/main/java/com/shippo/model/CustomsDeclaration.java

```diff
@@ -271,4 +273,12 @@ public Object getB13aNumber() {
 	public void setB13aNumber(Object b13aNumber) {
 		this.b13aNumber = b13aNumber;
 	}
+
+	public Object getInvoicedCharges() {
+        return invoicedCharges;
```

Comment:
is this another undocumented field?

Reply:
it's semi-documented... it's documented for the CustomsDeclaration object, but not for the POST payload in particular.  But it's a valid POST attribute.